apollo-router/src/query_planner/tests.rs

```diff
@@ -1877,23 +1877,17 @@ fn broken_plan_does_not_panic() {
             referenced_fields_by_type: Default::default(),
         }
         .into(),
-        query: Arc::new(Query::empty()),
+        query: Arc::new(Query::empty_for_tests()),
         query_metrics: Default::default(),
         estimated_size: Default::default(),
     };
     let subgraph_schema = apollo_compiler::Schema::parse_and_validate(subgraph_schema, "").unwrap();
     let mut subgraph_schemas = HashMap::default();
     subgraph_schemas.insert(
         "X".to_owned(),
-        query_planner::fetch::SubgraphSchema {
-            implementers_map: subgraph_schema.implementers_map(),
-            schema: Arc::new(subgraph_schema),
-        },
-    );
-    let result = Arc::make_mut(&mut plan.root)
-        .init_parsed_operations_and_hash_subqueries(&subgraph_schemas, "");
-    assert_eq!(
-        result.unwrap_err().to_string(),
-        r#"[1:3] Cannot query field "invalid" on type "Query"."#
```

Comment:
This seems to remove a test, is that intentional?

Reply:
Oh, thanks! This can be added back. For a few commits during our exploration/fixing cycles we did not return a Result from `init_parsed_operations_and_hash_subqueries`.